sections/sprint-1/pozyx.tex

```diff
@@ -0,0 +1,38 @@
+\section{Pozyx}
+Pozyx is a hardware/software solution that is used to provide positioning with an accuracy of down to 10 cm \cite{pozyx}.
+It makes use of ultra-wideband in combination with machine learning for positioning, which according to their documentation is more precise and efficient than traditional positioning systems such as WiFi, Bluetooth, RFID, and GPS.
+\\
+Since the two major requirements for positioning in this project are precision and a high update rate in order to ensure that the players can have reliable data available, the Pozyx system seems like a good place to start.
+\\
+For the update rate, the tags support update rates of up to 125 Hz for a single tag \cite{pozyx}.
+The Creator system from Pozyx is sold with 4 anchors and 5 locatable tags.
+An anchor is a stationary sensor used by the moveable tags to get their exact position.
+\subsection{Finding the location of anchors}
+For finding the position of a given tag using the anchors, a trilateration method is used.
+This method uses basic geometry to estimate the position by measuring the distance to the anchors of which we know the position.
+With this data, it is possible to draw a circle with a given radius.
+If we use two anchors, we will have two intersection points which are the possible positions of the tag.
+This means that to find a two-dimensional location, we will need at least three anchors, which will lead to only a single point where all three circles intersect.
```

Comment:
Overvejer om der måske skal noget paint i gang her for at vise konceptet??? Er der overskud???